stdlib/public/Darwin/Accelerate/vForce_Operations.swift

```diff
@@ -0,0 +1,1781 @@
+            precondition(bases.count == exponents.count && exponents.count == result.count)
+            
+            var n = Int32(bases.count)
+            
+            result.withUnsafeMutableBufferPointer { dest in
+                bases.withUnsafeBufferPointer { bases in
+                    exponents.withUnsafeBufferPointer { exponents in
+                        vvpow(dest.baseAddress!,
+                              exponents.baseAddress!,
+                              bases.baseAddress!,
+                              &n)
+                    }
+                }
+            }
+    }
+    
+    
+}
+
+// Array-Oriented Trigonometric Functions
+
+extension vDSP {
+    
+    // MARK: Sine
+    
+    /// Calculates the sine of each element in `vector`, writing the result to `result`, single-precision.
+    ///
+    /// - Parameter vector: Input values.
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func sin<U, V>(_ vector: U,
+                          result: inout V)
+        where
+        U : _ContiguousCollection,
+        V : _MutableContiguousCollection,
+        U.Element == Float, V.Element == Float {
+            
+            precondition(vector.count == result.count)
+            
+            var n = Int32(vector.count)
+            
+            result.withUnsafeMutableBufferPointer { dest in
+                vector.withUnsafeBufferPointer { src in
+                    vvsinf(dest.baseAddress!,
+                           src.baseAddress!,
+                           &n)
+                }
+            }
+    }
+    
+    /// Calculates the sine of each element in `vector`, writing the result to `result`, double-precision.
+    ///
+    /// - Parameter vector: Input values.
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func sin<U, V>(_ vector: U,
+                          result: inout V)
+        where
+        U : _ContiguousCollection,
+        V : _MutableContiguousCollection,
+        U.Element == Double, V.Element == Double {
+            
+            precondition(vector.count == result.count)
+            
+            var n = Int32(vector.count)
+            
+            result.withUnsafeMutableBufferPointer { dest in
+                vector.withUnsafeBufferPointer { src in
+                    vvsin(dest.baseAddress!,
+                          src.baseAddress!,
+                          &n)
+                }
+            }
+    }
+    
+    /// Calculates the sine of pi multiplied by each element in `vector`, writing the result to `result`, single-precision.
+    ///
+    /// - Parameter vector: Input values.
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func sinPi<U, V>(_ vector: U,
```

Comment:
This could also be spelled something like `sin(πTimes:)`.

Reply:
Also did `cosPi` and `tanPi`. I made the parameter name `piTimes` to avoid special characters.